miniirc.py

```diff
@@ -146,6 +146,9 @@ def ctcp(self, target, *msg, reply = False):
 
     def me(self, target, *msg):
         return self.ctcp(target, 'ACTION', *msg)
+    
+    def nick(new_nick):
```

Comment:
`irc.nick` is already used for the nick of the IRC client, and will override this function.

This function will need to have an extra `self` parameter as it is in a class.